runtime/tests/automl_runtime/sklearn/transformed_target_classifier_test.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2022 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import unittest
+from sklearn import datasets
+from sklearn.linear_model import LogisticRegression
+from sklearn.preprocessing import LabelEncoder
+
+from databricks.automl_runtime.sklearn import TransformedTargetClassifier
+
+
+class TestTransformedTargetClassifier(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Nit: I recommend using the regular `setUp` instead of `setUpClass`, for two reasons:

1. The mismatch of `cls.X` and `self.X` can be a little hard to read
2. Assume one test method modifies `self.X`, it might cause other test to fail if you use `setUpClass`, and that bug is hard to track down.

Reply:
yeah ideally the data should not be modified and that's why I put it in setUpClass, but you're right there's not a good way to enforce this immutability.